src/components/availability-label/helper.ts

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import { useGetAvailabilityV3 } from "../../core/fbs/fbs";
+import { useGetV1ProductsIdentifier } from "../../core/publizon/publizon";
+import { useConfig } from "../../core/utils/config";
+
+export const onlineMaterialAllowList = ["lydbog (net)"];
+
+export const isMaterialTypeOnline = (materialType: string) =>
+  onlineMaterialAllowList.includes(materialType);
```

Comment:
Please reconsider this.

Specifying internet audio books as the only online material type seems like a stretch. If it has been decided by the business that this is the way we do things for now then I guess it is OK but then it needs some kind of inline documentation explaining this.